.github/workflows/pr-run-recipes.yml

```
@@ -21,6 +21,27 @@ on:
           description: Applies best practices for Moderne Inc. SaaS repositories.
           recipeList:
             - org.openrewrite.java.testing.assertj.Assertj
+            #- org.openrewrite.java.testing.cleanup.BestPractices
```

```suggestion
```

maybe just remove it for now?

---

This is the better replacement if we don't want those assertDoesNotThrow introduced.
```suggestion
            - org.openrewrite.java.testing.cleanup.RemoveTestPrefix
```